index.js

```diff
@@ -110,8 +110,11 @@ var pluralTypeToLanguages = {
 
 function langToTypeMap(mapping) {
   var ret = {};
-  forEach(mapping, function (langs, type) {
-    forEach(langs, function (lang) {
+  objectEntries(mapping).forEach(function (entry) {
```

Comment:
The purpose of using `for-each` here is so this lib doesn't require `Array.prototype.forEach` to be available; this would be a breaking change.

Reply:
Should I use a for loop instead?

Reply:
i mean, if the goal is to modernize, a for loop wouldn't do it - if the goal is to remove a dependency, a for loop would do it, but it'd be nicer not to have to resort to that.

Reply:
My goal is to replace the for-each dependency with other dependencies that can be replaced with modern equivalents at the bundling level in the presence of a properly shimmed environment. The problem with for-each is that it can take either objects or arrays, so it can't be replaced by any standard thing automatically to reduce bundle sizes.